OOP/OOP7_StringFile/src/test/java/ReadFileTableTest.java

```diff
@@ -0,0 +1,14 @@
+import org.junit.Test;
+
+public class ReadFileTableTest {
+
+    @Test
+    public void processReadTableThreeColumn() throws Exception {
+        //given
+        ReadFileTable file = new ReadFileTable();
+        String path = "//home//lukasz//Pulpit//plik.txt";
```

Comment:
path otuside of project scope - won't work anywhere else, apart form your machine.
Please use relative path